### DISCUSSION TITLE
problem in installation of security onion

### DISCUSSION THREAD

**Discussion by sandraimmaculate** [Security-Onion-Solutions/securityonion#10835] [Unsupported Versions]
Hi, I'm studying on securityonin  I want to download security onion 2.3 in standalone node im also getting the same error. if i go and check in sosetup.log file the error was like (Command failed with exit code 1; will retry in 60 seconds (5 / 5)...
Command continues to fail; giving up.
Could not pull so-grafana:2.3.260, please ensure connectivity to ghcr.io
Errors detected during setup; skipping post-setup steps to allow for analysis of failures.) i also checked for connectivity its good 0% packet loss. when tried yesterday i got same error for wazuh now im getting for grafana.   if i execute so-mysql-start  there are 2 failures (cmd.run ID: so-mysql
    Function: cmd.run
        Name: until nc -z 10.10.20.0.0. 3306; do sleep 1; done
      Result: False
     Comment: Command "until nc -z 10.10.20.0.0. 3306; do sleep 1; done" run
     Started: 11:05:53.395104
    Duration: 600028.516 ms
     Changes:   
and  module.run ID: so-mysql
    Function: module.run
      Result: False
     Comment: One or more requisite failed: mysql.so-mysql
     Started: 11:15:53.437964
    Duration: 0.023 ms
     Changes:   )  i think because of this its not installing! what to do resolve this failure       can you please help me with this issues

**Comment by dougburks** [comment]
> I want to download security onion 2.3 in standalone node

What are the specs of your machine?
https://docs.securityonion.net/en/2.3/hardware.html#standalone-deployments

Did you verify the ISO image?

> Could not pull so-grafana:2.3.260, please ensure connectivity to ghcr.io

Do you have full Internet access?
https://docs.securityonion.net/en/2.3/firewall.html#internet-communication

**Reply by sandraimmaculate** [reply]
Thanks for replying and so sorry that was my mistake it's because of root volume limited storage space I solved it and  I installed Securityonion 2.3. May i know what is replacing Wazuh in Securityonion 2.4 Because i searched for it I didn't got any relevant answer for that

**Reply by dougburks** [reply]
Elastic Agent is replacing Wazuh in 2.4. For more information, please see the 2.4 documentation:
https://docs.securityonion.net/en/2.4/elastic-agent.html#elastic-agent

If you have further questions or problems, please start a new discussion since you've resolved your initial installation problem.

**Reply by sandraimmaculate** [reply]
thank you
